src/hyp3_sdk/hyp3.py

```diff
@@ -505,3 +505,12 @@ def check_quota(self) -> Union[float, int, None]:
         warn('This method is deprecated and will be removed in a future release.\n'
              'Please use `HyP3.check_credits` instead.', DeprecationWarning, stacklevel=2)
         return self.check_credits()
+
+    def costs(self) -> dict:
+        """
+        Returns:
+            Table of job costs
+        """
+        response = self.session.get(urljoin(self.url, '/costs'))
+        _raise_for_hyp3_status(response)
+        return response.json()
```

Comment:
Not an SDK issue per se, but the /costs json response and the costs() python response both render all the parameter values as strings, e.g.:
```
    'RTC_GAMMA': {
        'cost_parameter': 'resolution',
        'cost_table': {
            '10.0': 60,
            '20.0': 15,
            '30.0': 5,
        }
    }
```

That feels cumbersome to use in a python context. I'm not sure what we've gained by using the values as dictionary keys offsets the complexity of data type casting when comparing to an actual resolution value. Thoughts?

Reply:
The alternative requires looping instead of a simple dictionary lookup, but lets parameter values stay in their native data type:
```
    'RTC_GAMMA': {
        'cost_parameter': 'resolution',
        'cost_table': [
            {
                'parameter_value': 10.0,
                'cost': 60
            },
            {
                'parameter_value': 20.0,
                'cost': 15
            },
            {
                'parameter_value': 30.0,
                'cost': 5
            },
        ]
    }
```

Reply:
For a middle ground, could you do:
```
    'RTC_GAMMA': {
        'cost_parameter': 'resolution',
        'cost_table': [
            {
                'resolution': 10.0,
                'cost': 60
            },
            {
                'resolution': 20.0,
                'cost': 15
            },
            {
                'resolution': 30.0,
                'cost': 5
            },
        ]
    }
```
?

Reply:
Oh nevermind, I see the issue. My suggestion wouldn't help :thinking:

Reply:
I suppose the `HyP3.costs()` implementation stays the same regardless, so no need to bog down this PR.